simulationdataschema/model_system.py

```diff
@@ -551,19 +520,20 @@ def resolve_analyzed_atomic_cell(
 
         atomic_cell = AtomicCell(type=cell_type)
         atomic_cell.lattice_vectors = cell * ureg.angstrom
+        for label, atomic_number in zip(labels, atomic_numbers):
+            atom_state = AtomsState(chemical_symbol=label, atomic_number=atomic_number)
+            atomic_cell.m_add_sub_section(AtomicCell.atoms_state, atom_state)
```

Comment:
Maybe you can turn this into a 1-liner with `map`?

Reply:
I prefer to keep it this way, I think it is more readable.